test/e2e/deployment.go

```diff
@@ -654,11 +645,16 @@ func testRollbackDeployment(f *Framework) {
 	err = c.Extensions().Deployments(ns).Rollback(rollback)
 	Expect(err).NotTo(HaveOccurred())
 
+	// Wait for the deployment to start rolling back
```

Comment:
But that doesn't need to be done in this PR. I don't think I've ever seen it fail.